Simplified/Book/UI/NYPLBookDetailView.m

```diff
@@ -708,4 +721,28 @@ - (void)viewIssuesTapped {
   [self.detailViewDelegate didSelectViewIssuesForBook:self.book sender:self];
 }
 
+- (void)setUIAccessibilityFocusToView: (UIView *)view {
```

Comment:
wondering if these 3 functions could have been put into a swift extension instead.  Not a big deal -- an argument to keep them in objc is that the rest of the objc code requires these to even compile. 

But on the other hand writing objc code is adding to our tech debt. Generally speaking, I'd say if a piece of new code can be factored out into a swift extension pretty easily, I see more advantages than disadvantages. What do you think?

Reply:
You are right, it's better to have more swift code than Objc code. We are talking about an extension of `NYPLBookDetailView` in swift, right?

Reply:
right -- something like `NYPLBookDetailView+Accessibility`

Reply:
For `didFinishUIAccessibilityAnnouncementNotification:`, should we keep it in ObjC? Since it needs access to `normalView` and `buttonsView`

Reply:
i see what you mean… yeah, probably, since you would have to expose unneeded properties.